qcodes/dataset/sqlite_base.py

```diff
@@ -1363,6 +1374,7 @@ def get_parameter_data(conn: ConnectionPlus,
         output[output_param] = {name: np.array(column_data)
                          for name, column_data
                          in zip(param_names, res_t)}
```

Comment:
another comment unrelated to this PR: `res_t` can be left as a generator (without the outermost `list(...)`) since it's consumed by `zip`. probably, performance will be better.

Reply:
So I went and benchmarked this quickly, and the results are not conclusive. One method was not faster in all cases (somewhat surprisingly). But it *feels* faster to use a generator, so I'm gonna go with that. It was also fastest in most cases.